### PR TITLE
Use CRA (Redo): Pt 3/3: Working e2e tests

### LINKED ISSUE #212: Consider dropping Next.js
This is a low priority for now, but given the evolution of the project we may not really need Next.js, so we should consider dropping it once we get the MVP out of the way.

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR fixes e2e tests for CRA.**
- Also fixes all new eslint warnings.

Resolves #212 

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

**Comment by jessie-ross** [comment]
Looks like the e2e build is running out of memory, I'm going to look into our circleci settings to see if I can bump it.

**Comment by jessie-ross** [comment]
> Looks like the e2e build is running out of memory, I'm going to look into our circleci settings to see if I can bump it.

I fixed it by turning up the GC and increasing the timeout.

**Review by yanirs — APPROVED** [review]
Looks good, though we should look into the increased build time on CircleCI separately and `npm run dev` memory limit should be bumped before merging.

**Comment by yanirs** [comment]
Ah, and please change the Vercel setting before you merge the stack. :slightly_smiling_face:

File: package.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
     "*.{ts,tsx}": "eslint --fix"
   },
   "scripts": {
-    "build": "react-scripts build",
+    "build": "react-scripts --max_old_space_size=4096 build",
     "dev": "react-scripts start",
```
[the remark this answers is not in the document]
Comment (yanirs) [reply package.json:109]: Since this runs locally, it may be better to bump it even higher if it speeds things up.

**Pull request merged**